### PR TITLE
Introduced a metric "resp_time_ms" to understand how long the neon-proxy make a response

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**
Example from logs

`
2021-12-21 12:55:44,578 - pid:66 [D] handle_request:644 - >>> 140520376076032 0x7fcd731c4820 {"jsonrpc": "2.0", "id": 53, "result": "0x1a4113d1f89eeab26979240768d04d0757030ca398e38685a57d9104fd675f29"} eth_sendRawTransaction resp_time_ms= 469.0275192260742
`

### CODE REVIEW

File: proxy/plugin/solana_rest_api.py [hunk 1]
```diff
@@ -639,8 +640,10 @@ def handle_request(self, request: HttpParser) -> None:
             traceback.print_exc()
             response = {'jsonrpc': '2.0', 'error': {'code': -32000, 'message': str(err)}}
 
-        logger.debug('>>> %s 0x%0x %s %s', threading.get_ident(), id(self.model), json.dumps(response),
-                     request['method'] if 'method' in request else '---')
+        resp_time_ms = (time.time() - start_time)*1000  # convert this into milliseconds
```
Comment (gigimon) [inline proxy/plugin/solana_rest_api.py:643]: time.time() already have milliseconds
```python
In [2]: time.time()
Out[2]: 1640092880.2539148
```

File: proxy/plugin/solana_rest_api.py [hunk 2]
```diff
@@ -639,8 +640,10 @@ def handle_request(self, request: HttpParser) -> None:
...
+        logger.debug('>>> %s 0x%0x %s %s resp_time_ms= %s', threading.get_ident(), id(self.model), json.dumps(response),
+                     request['method'] if 'method' in request else '---',
```
Comment (gigimon) [inline proxy/plugin/solana_rest_api.py]: i think ternary operator we can change to:
```python
request.get('method', '---')
```

```python
In [1]: a = {}

In [2]: a.get('method', '---')
Out[2]: '---'
```

**Review by vodolaz — DISMISSED** [review]

**Review by gigimon — DISMISSED** [review]

File: proxy/plugin/solana_rest_api.py [hunk 3]
```diff
@@ -639,8 +640,10 @@ def handle_request(self, request: HttpParser) -> None:
             traceback.print_exc()
             response = {'jsonrpc': '2.0', 'error': {'code': -32000, 'message': str(err)}}
 
-        logger.debug('>>> %s 0x%0x %s %s', threading.get_ident(), id(self.model), json.dumps(response),
-                     request['method'] if 'method' in request else '---')
+        resp_time_ms = (time.time() - start_time)*1000  # convert this into milliseconds
+        logger.debug('>>> %s 0x%0x %s %s resp_time_ms= %s', threading.get_ident(), id(self.model), json.dumps(response),
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api.py:644]: Maybe better to print method and resp_time_ms before the response string? It is simple to get this info from logs.

File: proxy/plugin/solana_rest_api.py (hunk 1 above)
Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api.py:643]: It is not right. In seconds:
```
python3
Python 3.8.10 (default, Nov 26 2021, 20:14:08) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import time
>>> start_time = time.time()
>>> time.time() - start_time
11.44349479675293
```

File: proxy/plugin/solana_rest_api.py (hunk 3 above)
Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api.py:644]: It is not advisable to change the output format. It is preferable to add new information to the end of the line

File: proxy/plugin/solana_rest_api.py (hunk 2 above)
Comment (vasiliy-zaznobin) [reply proxy/plugin/solana_rest_api.py]: Ok. corrected

**Review by s-medvedev — APPROVED** [review]

File: proxy/plugin/solana_rest_api.py (hunk 1 above)
Comment (gigimon) [reply proxy/plugin/solana_rest_api.py:643]: @vasiliy-zaznobin  ok, in this case i agree

**Review by gigimon — APPROVED** [review]

**Pull request merged**